ng2-components/ng2-activiti-form/README.md

```diff
@@ -329,6 +348,70 @@ There are two additional functions that can be of a great value when controlling
 **Please note that if `event.preventDefault()` is not called then default outcome behaviour 
 will also be executed after your custom code.**
 
+## FormService
+
+```ts
+import { Component } from '@angular/core';
+import { FormService, FormEvent, FormFieldEvent } from 'ng2-activiti-form';
+
+@Component(...)
+class MyComponent {
+
+    constructor(private formService: FormService) {
+
+        formService.formLoaded.subscribe((e: FormEvent) => {
+            console.log(`Form loaded: ${e.form.id}`);
+        });
+
+        formService.formFieldValueChanged.subscribe((e: FormFieldEvent) => {
+            console.log(`Field value changed. Form: ${e.form.id}, Field: ${e.field.id}, Value: ${e.field.value}`);
+        });
+
+    }
+
+}
+```
+
+### Events
+
+| Name | Args Type | Description |
+| --- | --- | --- |
+| formLoaded | FormEvent | Raised when form has been loaded or reloaded |
+| formFieldValueChanged | FormFieldEvent | Raised when input values change |
+| taskCompleted | FormEvent | Raised when a task is completed successfully |
+| taskCompletedError | FormErrorEvent | Raised when a task is completed unsuccessfully  |
+| taskSaved | FormEvent | Raised when a task is saved successfully |
+| taskSavedError | FormErrorEvent | Raised when a task is saved unsuccessfully |
+
+### Methods
```

Comment:
yep understand....but we have to find an effective way to do that across all the projecr...can not be this service yes and another not. Olso maybe we need another place where to put the documentation for the service layer. Basically is that the conversation that I want to have:
- How we make sure that the documentation is the same for all the service
- Where to put this documentation
- Is it enough use JSDoc. ?